### ISSUE TITLE
Clean up exp:study-detail view

### ISSUE THREAD

**Issue by kimberscott** [lookit/lookit-api#518]
**TL;DR**: Clean up code for the experimenter-facing study detail view POST functions.

**Narrative**
It will be easier to reason about, maintain, and test this view if it is appropriately split up into pieces. 

**Acceptance Criteria**
- [ ] split up POST fn into 3 views: managing researcher permissions, changing study status, and cloning study. Perms on each should preserve the perms used in their current cases. Put processing of the comments field for managing study status in the appropriate view
- [ ] Tests of each POST view:
   - [ ] Can only be accessed as researcher with correct permissions for this study
   - [ ] Expected response / outcome. (Can mock / use eager evaluation for Celery tasks that get added, e.g. sending email to researcher upon being added to a study.)

See also https://github.com/lookit/lookit-api/issues/617 for related cleanup of the study edit view.
